### DISCUSSION TITLE
Icons in safari no resize with tailwind

### DISCUSSION THREAD

**Discussion by xumbreks** [lucide-icons/lucide#1688] [General]
I put a size on the icon with tailwindCSS and when I use safari to check the icon it doesn't change the size, but when I use chrome to check the size it changes normally. Can anyone tell me what could be wrong with safari ? or do you have any specific settings for safari? i'm using nextJS

**Comment by ericfennis** [comment]
Can you post a code example?

**Comment by heryckmp** [comment]
code example pls
